core/src/test/scala/com/colisweb/jrubysnesshours/core/ScheduleNextOpenTimeSpec.scala

```diff
@@ -0,0 +1,61 @@
+package com.colisweb.jrubysnesshours.core
+import java.time.LocalDate
+
+import org.scalatest.{Matchers, WordSpec}
+
+class ScheduleNextOpenTimeSpec extends WordSpec with Matchers {
+
+  import SpecUtils._
+
+  "Schedule.nextOpenTimeAfter" should {
```

Comment:
`"Schedule#nextOpenTimeAfter"` because it's an instance function, not a class function